gen/com/goide/psi/GoImportSpec.java

```diff
@@ -51,4 +51,6 @@
 
   boolean isCImport();
 
+  boolean isSDKImport();
```

Comment:
1. isSdkImport is not a property of import, so it shouldn't be instance method of ImportSpec. isCImport is a property because it depends on import spec and nothing more. Sdk- or not-sdk- import depends on module, settings of module, project, current context and many other things.

2. it's not good to introduce some method based on copy-paste from `GoPathScopeHelper` and do not patch `GoPathScopeHelper` itself.